test/models/prx_importer_test.rb

```diff
@@ -235,15 +247,23 @@ def self.new_guid
       doc = prx_importer.import(prx_story_id: 87683)
 
       doc.title.must_equal "Virginity, Fidelity, and Fertility"
-      doc.tags.sort.must_equal  ["PRX", "Weekly Program"]
+      doc.tags.sort.must_equal  ["Weekly Program"]
       doc.itags.sort.must_equal  ["prx:stories-87683", "prx_test"]
 
-      doc.links['profile'].href.must_match /story$/
+      doc.links['profile'].first.href.must_match /story$/
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.